packages/extension-api/src/extension-api.d.ts

```diff
@@ -1834,6 +1858,70 @@ declare module '@podman-desktop/api' {
     export const clipboard: Clipboard;
   }
 
+  /**
+   * Represents a dictionary object with string keys and values of type T.
+   */
+  export interface Dict<T> {
+    /**
+     * Index signature for accessing values by keys.
+     */
+    [key: string]: T | undefined;
+  }
+
+  /**
+   * Options for running a command.
+   */
+  export interface RunOptions {
+    /**
+     * Environment variables to set for the command.
+     */
+    env?: Dict<string>;
+  }
+
+  /**
+   * Represents the result of running a command.
+   */
+  export interface RunResult {
+    /**
+     * The exit code of the command.
+     */
+    exitCode: number;
+
+    /**
+     * The standard output (stdout) content of the command.
+     */
+    stdOut: string;
```

Comment:
probably need to use `stdout` (all lowercase instead of stdOut)

Reply:
removed the camelcase also for `stdErr`, now fields have name look like: `stdout` and `stderr`